providers/gitlab/resources/gitlab.go

```diff
@@ -228,12 +228,13 @@ func (p *mqlGitlabProject) protectedBranches() ([]interface{}, error) {
 		isDefaultBranch := branch.Name == defaultBranch
 
 		branchSettings := map[string]*llx.RawData{
-			"name":           llx.StringData(branch.Name),
-			"allowForcePush": llx.BoolData(branch.AllowForcePush),
-			"defaultBranch":  llx.BoolData(isDefaultBranch),
+			"name":              llx.StringData(branch.Name),
+			"allowForcePush":    llx.BoolData(branch.AllowForcePush),
+			"defaultBranch":     llx.BoolData(isDefaultBranch),
+			"CodeOwnerApproval": llx.BoolData(branch.CodeOwnerApprovalRequired),
```

Comment:
this needs to be lower case at the beginning

Reply:
done , Thanks